packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -38,9 +38,13 @@ const _removeGranuleFromCmr = async (granule, collectionId) => {
  * @param {Object} params
  * @param {Knex} params.knex - DB client
  * @param {Object} params.pgGranuleRecord - A Postgres granule record
- * @param {string} params.pgCollection - A Postgres Collection record
- * @param {Object} params.granulePgModel - Instance of granules model for PostgreSQL
- * @param {Object} params.granuleDynamoModel - Instance of granules model for DynamoDB
+ * @param {string} [params.pgCollection] - A Postgres Collection record
+ * @param {Object} [params.granulePgModel=new GranulePgModel()]
+ * - Instance of granules model for PostgreSQL
+ * @param {Object} [params.granuleDynamoModel=new models.Granule()]
+ * - Instance of granules model for DynamoDB
+ * @param {Function} [removeGranuleFromCmrFunction=_removeGranuleFromCmr()]
```

Comment:
```suggestion
 * @param {Function} [params.removeGranuleFromCmrFunction=_removeGranuleFromCmr()]
```

Reply:
This (and lower docstring comments) updated 3ab40c4345  -- some of the TS includes *do* obnoxiously break our valid-jsdoc linter, but it's a warning and allows ts-check to properly infer typing.     We should talk about  a better approach for the repo/process/etc.